modules/UI/UI.js

```diff
@@ -745,14 +745,19 @@ UI.setInitialMuteFromFocus = function (muteAudio, muteVideo) {
  * Mutes/unmutes the local video.
  */
 UI.toggleVideo = function () {
-    setVideoMute(!APP.RTC.localVideo.isMuted());
+    var newMuteState = !(APP.RTC && APP.RTC.localVideo && APP.RTC.localVideo.isMuted());
+    Analytics.sp.track('Video Mute Toggle', {
```

Comment:
`Analytics` is undefined. Arguably, we should be tracking this at the UI layer. @dansomething, has some cool angular directives that could be adapted to whatever analytics lib is pulled in at the app level.